Public/Src/Tools/ContentPlacementAnalysisTools/ContentPlacementAnalysisTools.Extraction/ContentPlacementAnalysisTools.Extraction.dsc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Tools.ContentPlacement.Extraction {
+
+    export declare const qualifier: BuildXLSdk.FullFrameworkQualifier;
+
+    // this is the log file config for cptools.builddownloader.exe
+    @@public

```

Comment:
once you add these files to the runtimeContent, you can reduce to visibily and make then just local by removing `@@public` and `export`